terraform/modules/paas/variables.tf

```diff
@@ -19,6 +19,10 @@ variable web_app_memory { default = 512 }
 variable worker_app_instances { default = 1 }
 variable worker_app_memory { default = 512 }
 
+variable logging {}
```

Comment:
This value can be calculated in the module, there is no need to pass it as an input, so we can simplify the interface.
Can we add a local variable `logging_service_name` instead?